app/src/components/Statistics/Statistics.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { StatisticsElement } from "./StatisticsElement";
+import brandIcon from "../../images/icon-brand-recognition.svg";
+import detailedIcon from "../../images/icon-detailed-records.svg";
+import fullyIcon from "../../images/icon-fully-customizable.svg";
+import { StyledWrapper, StyledHeader, StyledHeading, StyledDescription, StyledStatiscticsElements } from "./Statistics.styles";
+
+export const Statistics = () => {
+  const obj = [
```

Comment:
name "obj" means nothing. Your variable names should be more descriptive. Maybe try "Statistics data". Additionally, You keep the object inside the component and it is re-created every time the component renders. This is pointless. Move it out from the component. Maybe even to a separate file ;)

Reply:
StatisticsData*

Reply:
I moved it to seperate file and used useEffect and useState to fetch this only once during first render of the component.